wallet-new/src/Cardano/Wallet/API/V1/Headers.hs

```diff
@@ -2,10 +2,4 @@ module Cardano.Wallet.API.V1.Headers
     ( applicationJson
     ) where
 
-import           Network.HTTP.Types (Header, hContentType)
-
-
--- | Generates the @Content-Type: application/json@ 'HTTP.Header'.
-applicationJson :: Header
-applicationJson =
-    (hContentType, "application/json")
+import           Pos.Util.Servant (applicationJson)
```

Comment:
Sounds like we don't really need this module anymore :joy:

Reply:
Yeah, I'm just keeping it as a backwards compat shim so I don't have to touch N module import lists with a semantically null change.